scenarios/kubernetes_bazel.py

```diff
@@ -84,7 +84,15 @@ def query(kind, selected_pkgs, changed_pkgs):
 
     selection = '//...'
     if selected_pkgs:
-        selection = 'set(%s)' % ' '.join(selected_pkgs)
```

Comment:
I think this is more correct now, if a package has no prefix in build it is implicitly equivalent to `+//package/...` in a query, otherwise we just keep the operator as-is.

Reply:
(I did a lot of queries locally to verify this, though the docs seem to state that a build target list and a bazel query target list are equivilant...)

Reply:
https://docs.bazel.build/versions/master/user-manual.html#target-patterns
https://docs.bazel.build/versions/master/command-line-reference.html#target-pattern-syntax
https://github.com/kubernetes/test-infra/pull/5026#pullrequestreview-70358020

More specifically we only care if a target has a `-` prefix for build targets. For general bazel queries this is not implicit, and it's not clear that the build target patterns support any of the other query operators anyhow.